smtsolver.js

```diff
@@ -3,12 +3,13 @@ var execSync = require('child_process').execSync;
 var fs = require('fs');
 var tmp = require('tmp');
 
-const timeout = 10000;
+// Timeout in ms.
+const timeout = 1000;
 
 var potentialSolvers = [
   {
     name: 'z3',
-    params: '-smt2 rlimit=20000000 rewriter.pull_cheap_ite=true fp.spacer.q3.use_qgen=true fp.spacer.mbqi=false fp.spacer.ground_pobs=false'
+    params: '-smt2 timeout=' + timeout + ' rewriter.pull_cheap_ite=true fp.spacer.q3.use_qgen=true fp.spacer.mbqi=false fp.spacer.ground_pobs=false'
```

Comment:
~~Why time limit and not resource limit?~~

Ah this is just so that both cvc4/z3 have the same limit.